runtime/src/lib.rs

```diff
@@ -231,19 +231,38 @@ pub enum ProxyType {
     NonTransfer,
     Governance,
     Staking,
+    Vesting,
 }
 impl Default for ProxyType { fn default() -> Self { Self::Any } }
 impl InstanceFilter<Call> for ProxyType {
     fn filter(&self, c: &Call) -> bool {
         match self {
             ProxyType::Any => true,
             ProxyType::NonTransfer => !matches!(c,
-				Call::Balances(..) | Call::Indices(pallet_indices::Call::transfer(..))
+				Call::Balances(..) |
+				Call::Indices(pallet_indices::Call::transfer(..))
 			),
             ProxyType::Governance => matches!(c,
-				Call::Democracy(..) | Call::Council(..) | Call::Elections(..)
+				Call::Democracy(..) |
+				Call::Council(..) |
+				Call::Elections(..) |
+				Call::Utility(..)
 			),
-            ProxyType::Staking => matches!(c, Call::Staking(..)),
+            ProxyType::Staking => matches!(c,
+                Call::Staking(..) |
+                Call::Session(..) |
+				Call::Utility(..)
+            ),
+            ProxyType::Vesting => matches!(c,
+                Call::Staking(..) |
+                Call::Session(..) |
+                Call::Democracy(..) |
+				Call::Council(..) |
+				Call::Elections(..) |
+				Call::Vesting(pallet_vesting::Call::vest(..)) |
+				Call::Vesting(pallet_vesting::Call::vest_other(..)) |
+				Call::Utility(..)
```

Comment:
@mikiquantum so any transfer call will be filtered out if it's part of the utility batch?

Reply:
@vedhavyas actually looking at the code now, it seems that is only checking the parent call, so that would be a way to bypass a balance transfer. 
I will remove it to be safe.

Reply:
That means is something that can be bypassed in Kusama as well then.

Reply:
I think balance transfer would error out since the un-vested balances are still locked but since its the utility call, any call will filter through. 
Makes sense to remove it 👍🏼 @mikiquantum